templates/varnish.service.erb

```diff
@@ -41,10 +41,17 @@ ExecStart=/usr/sbin/varnishd <%= scope['::varnish::config::jail_opt'] %> \
   -p <%= k %>=<%= v %> \
 <% end -%>
   -S <%= scope['::varnish::secret_file'] %> \
-  -s <%= scope['::varnish::storage_type'] %>,<% if scope['::varnish::storage_type'] == 'file' -%><%= scope['::varnish::storage_file'] %>,<% end -%><%= scope['::varnish::storage_size'] %> \
-<% scope['::varnish::storage_additional'].each do |k| -%>
+  -s <%= scope['::varnish::storage_type'] %>,<% if scope['::varnish::storage_type'] == 'file' -%><%= scope['::varnish::storage_file'] %>,<% end -%><%= scope['::varnish::storage_size'] -%>
+<% unless [nil, :undefined, :undef, []].include?(scope['::varnish::storage_additional']) -%>
+ \
+<% end %>
```

Comment:
Should be `<% end -%>`

Reply:
@NITEMAN  added the missing - didn't see your suggest 2 Years ago :(